webgoat-lessons/jwt/src/main/java/org/owasp/webgoat/jwt/JWTQuiz.java

```diff
@@ -0,0 +1,49 @@
+package org.owasp.webgoat.jwt;
+
+import org.owasp.webgoat.assignments.AssignmentEndpoint;
+import org.owasp.webgoat.assignments.AttackResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+public class JWTQuiz extends AssignmentEndpoint {
+
+    private final String[] solutions = {"Solution 1", "Solution 2"};
```

Comment:
The solution for the second snippet is solution 3, not solution 2!

Reply:
or you should change the token on the page

Reply:
Fixed